src/javascripts/components/planes/planes.js

```diff
@@ -0,0 +1,13 @@
+const createPlane = (plane) => {
```

Comment:
call this `createPlaneCard` or something a little more descriptive to what it's doing